data.sql

```diff
@@ -5,3 +5,13 @@ insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) value
 insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Pikachu','7-Jan-2021',1,false,15.04);
 insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Devimon','12-May-2017',5,true,11);
 
+
+insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Charmander','8-Feb-2020',0,false,11);
+insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Plantmon','15-Nov-2021',2,true,5.7);
+insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Squirtle','2-Apr-1993',3,false,12.13);
+insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Angemon','12-Jun-2005',1,true,45);
+insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Boarmon','7-Jun-2005',7,true,20.4);
+insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Blossom','13-Oct-1998',3,true,17);
+insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Ditto','14-May-2022',4,true,22);
+
```

Comment:
- [ ] **Good job so far 👏 👏** However, I have noticed that you removed the *negative sign* ```-``` on the weight 😂, This could have been removed on this query ```Update all animals' weights that are negative to be their weight multiplied by -1.``` 

Kindly try to input data correctly